styles.css

```diff
@@ -79,13 +83,27 @@ label {
   visibility: hidden;
 }
 
+#clear-btn {
+  font-family: 'Quicksand', sans-serif;
+  font-size: 16px;
+  background: #134d71;
+  color: #ffffff;
+  display: block;
+  border: 0;
+  border-radius: 6px;
+  width: 50px;
+  height: 28px;
+  align-self: baseline;
+}
+
```

Comment:
I'm a little unclear about the end goal for how you want this button to be aligned. I'm not seeing a wireframe or spec for how it should look if you include a clear button in the project description. Do you have a wireframe or even a sketch that shows how you'd like the buttons to line up?

Some things that jump out as potentially problematic (aka things that can throw off alignment sometimes)
- `display: block`
- `align-self: baseline`

Again, hard to know without knowing what you want the end product to look like, so I'm totally happy to take another look later after you have a chance to clarify what you're trying to accomplish 😄